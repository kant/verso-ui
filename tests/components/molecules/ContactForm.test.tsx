import { jsx } from '@emotion/react'
import renderer from 'react-test-renderer'
import { ContactForm } from 'src/components'

test('Snapshot Test: Contact Form', () => {
  const component = renderer.create(
    <ContactForm>Heading Level One</ContactForm>,
  )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
