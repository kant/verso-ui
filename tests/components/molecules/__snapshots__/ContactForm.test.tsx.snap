// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Test: Contact Form 1`] = `
<form
  className="css-154v6jj-ContactForm"
>
  <div
    className="css-6n93mb-TextInput"
  >
    <label
      className="css-2x82w0-TextInput"
      htmlFor="First Name"
    >
      First Name
    </label>
    <input
      className="css-18ek46i-TextInput"
      id="First Name"
      name="First Name"
      type="text"
    />
  </div>
  <div
    className="css-1n3bh1d-TextInput"
  >
    <label
      className="css-2x82w0-TextInput"
      htmlFor="Last Name"
    >
      Last Name
    </label>
    <input
      className="css-18ek46i-TextInput"
      id="Last Name"
      name="Last Name"
      type="text"
    />
  </div>
  <div
    className="css-boj1sm-TextInput"
  >
    <label
      className="css-2x82w0-TextInput"
      htmlFor="Email Address"
    >
      Email Address
    </label>
    <input
      className="css-18ek46i-TextInput"
      id="Email Address"
      name="Email Address"
      type="email"
    />
  </div>
  <div
    className="css-1agg0r2-TextInput"
  >
    <label
      className="css-2x82w0-TextInput"
      htmlFor="Subject"
    >
      Subject
    </label>
    <input
      className="css-18ek46i-TextInput"
      id="Subject"
      name="Subject"
      type="text"
    />
  </div>
  <div
    className="css-12anhju-TextArea"
  >
    <label
      className="css-1bawu2g-TextArea"
      htmlFor="Message"
    >
      Message
    </label>
    <textarea
      className="css-1jf3zmb-TextArea"
      id="Message"
      name="Message"
    />
  </div>
  <button
    className="css-1195o8y-PrimaryButton"
    type="button"
  >
    Send
  </button>
</form>
`;
